docs/large-bucket/README.md

```diff
@@ -1,13 +1,10 @@
 # Large Bucket Support Quickstart Guide [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io) [![Go Report Card](https://goreportcard.com/badge/minio/minio)](https://goreportcard.com/report/minio/minio) [![Docker Pulls](https://img.shields.io/docker/pulls/minio/minio.svg?maxAge=604800)](https://hub.docker.com/r/minio/minio/) [![codecov](https://codecov.io/gh/minio/minio/branch/master/graph/badge.svg)](https://codecov.io/gh/minio/minio)
 
-Minio erasure code limits the number of disks in a deployment to 16. This allows enough storage space for a single deployment to hold a tenant's data. However, there are use cases which 
-may need larger number of disks/storage space upfront. Keeping this in consideration, we added large bucket support.
+Minio erasure code limits the number of disks in a deployment to 16. This allows enough storage space for a single deployment to hold a tenant's data. However, there are use cases which may need larger number of disks/storage space upfront. Keeping this in consideration, we added large bucket support.
```

Comment:
Why say this when it is no longer true:
`Minio erasure code limits the number of disks in a deployment to 16.`

Something like this seems better:
`Erasure code in Minio is limited to 16 disks - this on its own allows storage space to hold a tenant's data. However, to cater to cases which may need larger number of disks or high storage space requirements, large bucket support was introduced.`